cmd/build/command.go

```diff
@@ -164,19 +165,21 @@ func (*Command) loadContext(ctx context.Context, options *types.BuildOptions) er
 	// information cannot be extracted so call to GetContextResource is skkiped.
 	if err := validateDockerfile(options.File); err != nil {
 		ctxResource, err := model.GetContextResource(options.File)
-		if err != nil {
+		if err != nil && !errors.Is(err, discovery.ErrOktetoManifestNotFound) {
 			return err
 		}
 
-		if err := ctxResource.UpdateNamespace(options.Namespace); err != nil {
-			return err
-		}
-		ctxOpts.Namespace = ctxResource.Namespace
+		if ctxResource != nil {
```

Comment:
Can we include a comment to specify why if the `ctxResource` is nil we keep going? There are some hidden knowledge behind this